x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/osquery/index.tsx

```diff
@@ -83,6 +88,17 @@ const OsqueryEditorComponent = ({
     [onSave]
   );
 
+  const noOsqueryPermissions = useMemo(
+    () =>
+      (!osqueryPermissions.runSavedQueries || !osqueryPermissions.readSavedQueries) &&
```

Comment:
Just a question. Can a user ever `write` but not `read` or `run`? Just curious what the real-life permission combinations are?